spec/src/main/asciidoc/configuration.asciidoc

```diff
@@ -412,7 +412,7 @@ Please see <<verification-publickey-location, mp.jwt.publickey.location>> for al
 #### `mp.jwt.decrypt.key.algorithm`
 
 The `mp.jwt.decrypt.key.algorithm` configuration property allows for specifying which key management key algorithm
-is supported by the MP JWT endpoint. Algorithms which must be supported are either `RSA-OAEP` or `RSA-OAEP-256`. Default value is `RSA-OAEP` but `RSA-OAEP-256` will become a default value in one of the next major releases.
+is supported by the MP JWT endpoint. Algorithms which must be supported are either `RSA-OAEP` or `RSA-OAEP-256`. If `mp.jwt.decrypt.key.algorithm` is not supplied, the value defaults to a vendor-specific behavior as was the case for MP-JWT 2.0.
```

Comment:
Given that RSA-OAEP is only recommended for backwards compatibility in https://www.rfc-editor.org/rfc/rfc8017#section-7.1.1 due to the use of SHA-1, defaulting to RSA-OAEP does not seem wise. I would agree that allowing the vendor to default to whatever they view as reasonable makes sense.

Reply:
The section "Required and Recommended MP-JWT Headers and Claims" (interoperability.asciidoc) still says that _RSA-OAEP_ is required as alg value which contradicts the possibility to have _RSA-OAEP-256_.

Reply:
@rdebusscher I've updated the interoperability.asciidoc so it aligns with the description of `mp.jwt.decrypt.key.algorithm`.  Specifically it says the values can be `RSA-OAEP` or `RSA-OAEP-256` and that values such as `RSA-OAEP-384`, `RSA-OAEP-512` and others are optional.  I've done the same with the `alg` description.

As this section is on interoperability, one potential improvement might be to explicitly say "are optional and may not be portable"

Thoughts?